ar4_hardware_interface/urdf/ar_gripper.ros2_control.xacro

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
```

Comment:
We should change the name of this file `gripper.ros2_control.xacro` to match with the name used in Gazebo.

Reply:
Done.